assembl/static2/js/app/components/login/signupForm.jsx

```diff
@@ -161,6 +163,7 @@ const mapDispatchToProps = dispatch => ({
 
 const withData = graphql(TabsConditionQuery, {
   props: ({ data }) => ({
+    ...data,
     hasTermsAndConditions: data.hasTermsAndConditions
```

Comment:
Good! Please try to use a more "signifiant" commit message next time (sometimes you need to find a commit and it's much easier if it has a specific message)